1.0/resources/tools.md

```diff
@@ -0,0 +1,222 @@
+    </head>
+
+For more documentation, see [http://polygit.org](http://polygit.org/).
+
+Source: [github.com/PolymerLabs/polygit](https://github.com/PolymerLabs/polygit)
+
+## Development tools {#dev}
+
+### <b>&lt;seed-element></b> - boilerplate starter for creating a resuable element on Github {#seedelement}
+
+See [Create a reusable element](/1.0/docs/start/reusableelements.html).
+
+### <b>Polymer Starter Kit</b> - starter project for building Polymer apps {#psk}
+
+The Polymer Starter Kit is a complete starting point for building Polymer-based applications See [Using Polymer Starter Kit](/1.0/docs/start/psk/set-up.html) for more information on getting started.
+
+Source: [github.com/PolymerElements/polymer-starter-kit](https://github.com/PolymerElements/polymer-starter-kit)
+
+### <b>polylint</b> - lint projects for errors and common mistakes {#polylint}
+
+[polylint](https://github.com/PolymerLabs/polylint) is a linting tool to detect
+common mistakes and errors in your projects.
+
+Install:
+
+    npm install -g polylint
+
+Usage:
+
+    polylint --root my-project/ \
+        --input elements.html my-element.html
+
+**Source:** [github.com/PolymerLabs/polylint](https://github.com/PolymerLabs/polylint)
+
+**Related tools**
+
+- [Sublime plugin](https://github.com/nomego/SublimeLinter-contrib-polylint)
+- [Atom package](https://github.com/PolymerLabs/polylint#installing-the-atom-package)
+
+### <b>polystyle</b> - web service for creating style modules {#polystyle}
+
+[polystyle](https://poly-style.appspot.com/demo/) is a web service that can wrap an existing stylesheet on a remote server as Polymer [style module](https://www.polymer-project.org/1.0/docs/devguide/styling.html#style-modules). This is useful if you have a hosted third-party stylesheet that you want to use in an element or your application.
+
+Usage:
+
+    <head>
+      <link rel="import" href="bower_components/polymer/polymer.html">
+      <link rel="import" href="https://poly-style.appspot.com?id=theme-styles&url=https://example.com/styles.css">
+      <style is="custom-style" include="theme-styles">
+        ...
+      </style>
+    </head>
+
+For more information, see [https://poly-style.appspot.com/demo/](https://poly-style.appspot.com/demo/).
+
+**Related tools**
+
+- [gulp-style-modules](https://github.com/MaKleSoft/gulp-style-modules) - 3rd party Gulp plugin for wrapping local CSS files into style modules
+
+Source: [github.com/PolymerLabs/polystyles](https://github.com/PolymerLabs/polystyles)
+
+## Testing / debugging {#testing}
+
+### Web component tester - unit-testing tool for elements {#wct}
+
+[Web component tester](https://github.com/Polymer/web-component-tester) is a tool providing a browser-based testing environment for web components. Out of the box it includes support for Mocha, Chai, Async and Sinon. Read more about it in "Unit Testing Polymer Elements](/articles/unit-testing-elements.html)".
```

Comment:
Kayce is working on a new article for this. https://github.com/Polymer/docs/pull/1474